### PR TITLE
fix: adjust the profile context

### PR DESCRIPTION
**Pull request by alanhoskins**
Putting in a couple separate PRs. This one slightly reworked the useProfileContext/ProfileContext, to store the profile data along with the profile ref. This also includes #103 already merged in, so that one might want to be merged in first. 

I still have to add in validation on continue/next button.

### CODE REVIEW

File: src/components/layout/BlobBox.tsx [hunk 1]
```diff
@@ -64,4 +31,38 @@ const BlobBox = (props: BlobBoxProps) => {
...
+        }
+      `
+  );
+
+const Blobs = styled.div<{ blobs: any }>`
+  display: none;
+  z-index: -1;
+  transform-style: preserve-3d;
+
+  @media (min-width: 768px) {
```
Comment (bestfoodalex) [inline src/components/layout/BlobBox.tsx:56]: We may want to add breakpoint style variables

File: src/components/layout/BlobBox.tsx [hunk 2]
```diff
@@ -64,4 +31,38 @@ const BlobBox = (props: BlobBoxProps) => {
...
+    display: block;
+  }
+
+  img {
+    position: absolute;
+    overflow: visible;
```
Comment (bestfoodalex) [inline src/components/layout/BlobBox.tsx:62]: Why is there an overflow property on the img tag?

**Review by bestfoodalex — APPROVED** [review]

File: src/components/layout/BlobBox.tsx (hunk 1 above)
Comment (alanhoskins) [reply src/components/layout/BlobBox.tsx:56]: Not sure. I didn't write the initial component, just moved it out of the render so the console did not spit out warnings/errors. I can look at removing it though.

File: src/components/layout/BlobBox.tsx (hunk 2 above)
Comment (alanhoskins) [reply src/components/layout/BlobBox.tsx:62]: Not sure. I didn't write the initial component, just moved it out of the render so the console did not spit out warnings/errors. I can look at removing it though.

Comment (bestfoodalex) [reply src/components/layout/BlobBox.tsx:62]: @alanhoskins we can remove it later 👍🏻

**Pull request merged**